js/script.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-unused-vars */
+
+const booksElement = document.querySelector('.books');
+const titleInput = document.querySelector('.title-input');
+const authorInput = document.querySelector('.author-input');
+const addBookBtn = document.querySelector('#addBtn');
+
+let books = [];
+
+const saveToLocalStorage = (books) => {
+  localStorage.setItem('books', JSON.stringify(books));
+};
+
+const getExistingBooks = () => JSON.parse(localStorage.getItem('books'));
+
+const addBooks = () => {
```

Comment:
- [ ] It will be nice to uniquely generate an `id` attribute for each book before persisting it to the local storage.

Reply:
Hmm... This seems like a better approach